test/cli/commands_test.py

```diff
@@ -678,33 +678,24 @@ def test__cli__fix_error_handling_behavior(sql, fix_args, fixed, exit_code, tmpd
             assert not fixed_path.is_file()
 
 
-_old_crawl = BaseRule.crawl
```

Comment:
Had to update this test now that `crawl()` is no longer called recursively. It now mocks the `_eval()` function instead.